src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius, String color) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public double getRadius() {
+        return radius;
+    }
+
+    public void setRadius(double radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double area() {
+        return radius * radius * Math.PI;
+    }
+
+    @Override
```

Comment:
Don't put all behavior into a single class/interface if the methods are conceptually different. The `draw()` method should not be in the same interface as `area()` if they are conceptually different.